tabs-tabs-tabs/tabs.js

```diff
@@ -124,3 +124,21 @@ document.addEventListener("click", function(e) {
 
   e.preventDefault();
 });
+
+//onRemoved listener. fired when tab is removed
+chrome.tabs.onRemoved.addListener(function(tabId, removeInfo){
+  console.log(`The tab with id: ${tabId}, is closing`);
+
+  if(removeInfo.isWindowClosing) {
+    console.log(`Its window is also closing.`);
+  } else {
+    console.log(`Its window window is not closing`);
+  }
+});
+
+//onMoved listener. fired when tab is moved into the same window
+chrome.tabs.onMoved.addListener(function(tabId, moveInfo){
+  var startIndex = moveInfo.fromIndex;
+  var endIndex = moveInfo.toIndex;
+  console.log(`Tab with id: ${tabId} moved from index: ${startIndex} to: ${endIndex}`);
```

Comment:
Super picky, but since you'll need to fix the typo above anyway...

...this gives output like:

`Tab with id: 4 moved from index: 2 to: 0`

I thing it would be better to repeat "index", like:

`Tab with id: 4 moved from index: 2 to index: 0`